SPECS/node-problem-detector/node-problem-detector.spec

```diff
@@ -40,9 +38,9 @@ Default configuration files for node-problem-detector
 %install
 mkdir -p %{buildroot}%{_bindir}/
 install -vdm 755 %{buildroot}/%{_bindir}
-install -pm 755 output/linux_amd64/bin/node-problem-detector %{buildroot}%{_bindir}/
```

Comment:
Totally cool with this change, just doing a double check here to make sure that anyone on AMD64 that upgrades to this version will not be impacted.  (I don't imagine there will be any...just want to make sure we've thought it through)

Reply:
Thanks for the check, I have confirmed that this is the same binary from the previous version/release. The patch introduced in this PR edits the path to the binary in the build environment to reflect the multi-arch nature of this update.